tests/setuponlytests/vanilla_world_for_spigot_server/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
+
+services:
+  mc:
+    restart: "no"
+    image: ${IMAGE_TO_TEST:-itzg/minecraft-server}
+    environment:
+      EULA: "TRUE"
+      SETUP_ONLY: "TRUE"
+      VERSION: ${MINECRAFT_VERSION:-LATEST}
+      TYPE: "PAPER"
```

Comment:
Noticed in the folder name you called out spigot, but see you are using Paper. Probably just a typo or me not knowing enough about what is going on. :)

Reply:
I'm mostly protecting against the possibility of a future refactor accidentally making the Spigot logic conditional on `TYPE` (i.e. apply only to Spigot itself) instead of `FAMILY` (apply to all Spigot-style jars). It felt like overkill to write a whole separate test for it, but might as well catch it here.